WcaOnRails/app/models/registration.rb

```diff
@@ -335,6 +335,14 @@ def series_registration_info
                                    .join(" + ")
   end
 
+  # NOTE: This is really bad, only for a proof of concept. It assumes that there is only ONE PaypalTransaction and ONE PaypalCapture - not safe assumptions.
+  def get_capture_id
```

Comment:
In Ruby, convention dictates that method names are never prefixed with `get_` or `set_`.
This should just be called `def capture_id`